src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -75,36 +75,42 @@ public static HearingDetails buildHearingDetails(HearingWrapper wrapper, Referen
 
     public static boolean shouldBeAutoListed(@Valid SscsCaseData caseData) {
         // TODO Future Work
-        return !isCaseLinked(caseData);
+        return !isCaseLinked(caseData) && !isCaseUrgent(caseData);
+    }
+
+    public static boolean isCaseUrgent(@Valid SscsCaseData caseData) {
+        return isYes(caseData.getUrgentCase());
     }
 
     public static String getHearingType() {
         return SUBSTANTIVE.getHmcReference();
     }
 
     public static HearingWindow buildHearingWindow(@Valid SscsCaseData caseData, boolean autoListed) {
-
-        if (!autoListed || isBlank(caseData.getDwpResponseDate())) {
-            LocalDate dateRangeStart = LocalDate.now().plusDays(DAYS_TO_ADD_HEARING_WINDOW_TODAY);
-            return HearingWindow.builder()
-                    .dateRangeStart(dateRangeStart)
-                    .dateRangeEnd(null)
-                    .build();
-        }
-        LocalDate dwpResponded = LocalDate.parse(caseData.getDwpResponseDate());
-
-        LocalDate dateRangeStart = isYes(caseData.getUrgentCase())
-                ? dwpResponded.plusDays(DAYS_TO_ADD_HEARING_WINDOW_DWP_RESPONDED_URGENT_CASE)
-                : dwpResponded.plusDays(DAYS_TO_ADD_HEARING_WINDOW_DWP_RESPONDED);
-
-
         return HearingWindow.builder()
                 .firstDateTimeMustBe(getFirstDateTimeMustBe())
-                .dateRangeStart(dateRangeStart)
+                .dateRangeStart(getHearingWindowStart(caseData, autoListed))
                 .dateRangeEnd(null)
                 .build();
     }
 
+    public static LocalDate getHearingWindowStart(@Valid SscsCaseData caseData, boolean autoListed) {
```

Comment:
Minor: we can return these inline rather than assigning to startDate - would make the code a lot cleaner

Reply:
Refactored.
Left startDate assignment as it's returned twice, after 1st condition check and as default at the end, but just returned inline for conditions 2 and 3.